### DISCUSSION TITLE
[Feature] replace upload-button with "scan" (for read-only external library)

### DISCUSSION THREAD

**Discussion by poisonnuke** [immich-app/immich#11705] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In scenarios, where the only source for pictures is an external library (that is set to read-only), the upload button does not make any sense (and should be removed to improve UX).

at the same time users need to constantly go into administration and click on scan new files to update the library in case of a recent addition of media files (and they do not want to wait for the background-scan to start)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
External libraries getting special treatment has been a source of complexity and problems, so we're trying to move away from that. Thus I don't think we'll be doing anything like this.
